rrule/src/wasm/mod.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(
+    clippy::cast_possible_truncation,
+    clippy::doc_markdown,
+    clippy::unwrap_used
+)]
+
+use wasm_bindgen::prelude::*;
+
+use crate::{RRuleSet, RRuleError};
+
+/**
+  Get all recurrences of the rrule!
+     rule_set_string: List of rrules
+     limit: Limit must be set in order to prevent infinite loops
+*/
```

Comment:
Rust uses triple slashes for docs, ref: https://doc.rust-lang.org/rust-by-example/meta/doc.html